actions/info.action.ts

```diff
@@ -21,6 +21,11 @@ interface PackageJsonDependencies {
 interface NestDependency {
   name: string;
   value: string;
+  fullName: string;
```

Comment:
can we rename `fullName` to `packageName`?

Reply:
done